velox/exec/tests/utils/PlanBuilder.cpp

```diff
@@ -99,13 +99,11 @@ PlanBuilder& PlanBuilder::tableScan(
     const RowTypePtr& dataColumns,
     const std::unordered_map<
         std::string,
-        std::shared_ptr<connector::ColumnHandle>>& assignments,
-        bool isFilterPushdownEnabled) {
+        std::shared_ptr<connector::ColumnHandle>>& assignments) {
```

Comment:
Is removing isFilterPushdownEnabled parameter related to the Timestamp reader?

Reply:
Thanks for your review. To add `isFilterPushdownEnabled` parameter was a temporary change before supporting the filter pushdown of Timestamp. After its support in [Filter.h](https://github.com/facebookincubator/velox/pull/4680/files#diff-280c77e7a8b1d7cfaeaaed70f8ac11c67e881b4f1682ee099bc326a1b53e9349R1826-R1829), this change has been removed from this PR.